internal/consensus/byzantine_test.go

```diff
@@ -276,6 +276,8 @@ func TestByzantinePrevoteEquivocation(t *testing.T) {
 	pubkey, err := bcs.privValidator.GetPubKey()
 	require.NoError(t, err)
 
+	const timeout = 30 * time.Second // Increase timeout to 30 seconds
```

Comment:
Does this reduce the frequency of test failures?

Reply:
yes, I think there was a performance aspect to it